otelfiber/fiber.go

```diff
@@ -32,28 +30,28 @@ const (
 )
 
 // Middleware returns fiber handler which will trace incoming requests.
-func Middleware(service string, opts ...Option) fiber.Handler {
-	if service == "" {
-		service = defaultServiceName
-	}
+func Middleware(opts ...Option) fiber.Handler {
```

Comment:
this is a breaking change or ?

Reply:
> this is a breaking change or ?

Yes, this is definitely a breaking change. It's being made as `http.server_name` is no longer listed as an attribute in the semantic conventions for either HTTP metrics and/or spans. As mentioned,

> I have tried to retain the attribute for those that depend on it, however I'm not sure how everyone will feel about the change in the function signature?

FYI: If any consumers do depend on the attribute, it can be retained going forward by making use of the `WithServerName` option as shown in the tests at https://github.com/gofiber/contrib/blob/f3bf30b3165d1aaf29e501e54c6ca640687bbc04/otelfiber/fiber_test.go#L73-L78

Happy to discuss alternatives/options if you want to avoid this.